schemas/linkedDataSignature2020.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const w3cDateTime = require('./w3cDateTime');
+const identifier = require('./identifier');
+
+const signature = {
+  title: 'An Ed25519 Linked Data Signature',
```

Comment:
This should either be something like "2020 Linked Data Signatures" and allow for more than one suite, or it should just be named `Ed25519Signature2020`, right?

Reply:
Renamed to Ed25519Signature2020.